src/org/zaproxy/zap/GuiBootstrap.java

```diff
@@ -68,16 +68,16 @@
  * @since 2.4.2
  */
 public class GuiBootstrap extends ZapBootstrap {
-
-    private final Logger logger = Logger.getLogger(GuiBootstrap.class);
-
-    /**
-     * Flag that indicates whether or not the look and feel was already set.
-     * 
-     * @see #setupLookAndFeel()
-     */
+	
+    private final static Logger logger = Logger.getLogger(GuiBootstrap.class);
+    
+    /**	
+    -     * Flag that indicates whether or not the look and feel was already set.	
+    -     * 	
+    -     * @see #setupLookAndFeel()	
+    -     */	
     private boolean lookAndFeelSet;
-
+    private static boolean isLookAndFeelSet;
```

Comment:
This should be a local variable (in fact the method could return immediately, without using an auxiliary variable).

Reply:
@thc202 I tried something like that. Instead of using an auxiliary variable, the method return true. However, when ZAP launches for the first time, the default look is metal instead of nimbus. So,I resorted to using an auxiliary variable.

Reply:
Odd, did it return true just when set?

Reply:
@thc202 Wait,I think I have figured out my mistake. I have submitted a new commit where there is no use of local variable.

Reply:
Cool :)